app/views/core/SingletonAppVueComponentView.js

```diff
@@ -24,7 +24,12 @@ export default class SingletonAppVueComponentView extends VueComponentView {
       store,
       router: this.router,
 
-      render: (h) => h(Root)
+      render: (h) => h(Root),
+
+      provide: {
```

Comment:
This is so cool. Wow.

Reply:
Yes agreed!  I learned about this too when trying to find ways to expose the CocoView methods to Vue, really excited about it, super clean.